src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--sun calendar--29">
+      <div class="day-1"></div>
```

Comment:
```suggestion
      <div class="day"></div>
```
Не надо уникальный класс каждому элементу задавать, используй nth-child, если надо стилизовать элемент в зависимости от позиции